randovania/games/cave_story/logic_database/Grasstown.txt

```diff
@@ -156,48 +156,38 @@ Extra - starting_script: <TRA0006:0098:0004:0018
       Trivial
   > Pickup (Kazuma Chest)
       Rusty Key
-  > Area Centre
-      All of the following:
-          Any of the following:
-              # Get up to the central area
-              After Activated Fans or Has Flight
-              After Entered Grasstown from Fireplace and Remove Points of No Return
-              Any of the following:
-                  # Reverse Chaco Skip
-                  # TAS Jump
-                  Gravity Hopping (Hypermode)
-                  # https://www.youtube.com/watch?v=WiA3lsW5gec
-                  Damage Boosting (Advanced) and Normal Damage ≥ 2
-                  # https://www.youtube.com/watch?v=rmtwHNrxU5Q
-                  Damage Boosting (Intermediate) and Normal Damage ≥ 3
-          Any of the following:
-              # Get past the Mannan
-              Has Weapon
-              All of the following:
-                  Pacifist Strats (Intermediate)
-                  Any of the following:
-                      Has Flight
-                      Damage Boosting (Beginner) and Normal Damage ≥ 3
-              Gravity Hopping (Advanced) and Pacifist Strats (Advanced)
   > Event - Saved Kazuma
       Explosive
   > Event - Level MG (East)
       Machine Gun
   > Near Hint
       Trivial
+  > Beyond First Mannan
+      Any of the following:
+          After Activated Fans or Has Flight
+          After Entered Grasstown from Fireplace and Remove Points of No Return
+          Any of the following:
+              # Reverse Chaco Skip
+              # https://www.youtube.com/watch?v=rmtwHNrxU5Q
+              Damage Boosting (Intermediate) and Normal Damage ≥ 3
+              # https://www.youtube.com/watch?v=WiA3lsW5gec
+              Damage Boosting (Advanced) and Normal Damage ≥ 2
```

Comment:
some other DBs have the habit of having something like `3 hit method: <link>`, `2 hit method: <link>`
Your choice if you want to do it here too, wanted to at least point it out.

Reply:
I've done that for connections with good names like that. for this connection I distinguish the methods by their difficulty anyway so it would be redundant